Open-Rival/include/UnitAnimationLookup.h

```diff
@@ -0,0 +1,179 @@
+#ifndef UNIT_ANIMATION_LOOKUP_H
+#define UNIT_ANIMATION_LOOKUP_H
+
+#include <map>
+
+namespace Rival {
+
+    enum class UnitType : int {
```

Comment:
I think it makes more sense for this to live in `Unit.h` as it is not only relevant to the animation. It will also determine a unit's stats, abilities, etc.